cypress/integration/channel/message_draft_with_attachment_then_switch_channel_sepc.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+describe('Message Draft with attachment and Switch Channels', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M14126 Message Draft Pencil Icon - No text, only file attachment', () => {
+        // # Got to a test channel on the side bar
+        cy.get('#sidebarItem_town-square').scrollIntoView();
+        cy.get('#sidebarItem_town-square').should('be.visible').click();
+
+        // * Validate if the channel has been opened
+        cy.url().should('include', '/channels/town-square');
+
+        // * Validate if the draft icon is not visible on the sidebar before making a draft
+        cy.get('#sidebarItem_town-square').scrollIntoView();
```

Comment:
This may not impact your test, but I've been cleaning up a bunch of other tests and, depending on the state and data that is in your environment, scrollIntoView() for town square may put it into a position where it is covered by the "more messages" bubble. So, I've been doing `cy.get('#publicChannel').scrollIntoView()` instead to scroll the header into view, and then the channels should be visible.